CMakeSettings.json

```diff
@@ -1,27 +0,0 @@
-﻿{
```

Comment:
Why did you delete this file? It's how I'm able to run in both debug and release mode. Of course I can have a local copy, but I think other people were using it as well.